.github/workflows/pre-commit.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v3
+    - name: Get modified files
+      id: modified-files
+      run: echo "modified_files=$(git diff --name-only -r HEAD^1 HEAD | xargs)" >> $GITHUB_OUTPUT
```

Comment:
Looks like the step to get changed files isn't working correctly. Looking into it

Reply:
I think I've fixed it. Just need to launch the pipeline to confirm.

Reply:
What happen if that is a deleted file?

Reply:
Nothing - `pre-commit` simply skips those:
```
λ pre-commit run --files nonexistent
isort................................................(no files to check)Skipped
...
Add License..........................................(no files to check)Skipped
```